nautobot_device_lifecycle_mgmt/migrations/0031_cvelcm_last_modified_date.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.21 on 2024-02-27 01:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("nautobot_device_lifecycle_mgmt", "0030_delete_models_migrated_to_core"),
```

Comment:
We'll need to rename/regenerate this. I had to pull in migration for Nautobot 2.3 compatibility into next3.0

Reply:
This has been regenerated.

Reply:
This is now resolved.